.github/pull_request_template.md

```diff
@@ -16,7 +16,7 @@ Please use this format to link other issues with their numbers: Close #123
 https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
 -->
 
-Close #
+Closes #
```

Comment:
actually `close` also sounds sensible tbh, we want the PR to close that particular issue on merging, what do you think?

Reply:
I think both are okay but I felt like `this PR closes this issue` sounds better + I see that most of the repos use `Closes` or `Fixes` in the PR template [pandas](https://github.com/pandas-dev/pandas/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [dask](https://github.com/dask/dask/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [pytorch](https://github.com/pytorch/pytorch/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [jina](https://github.com/jina-ai/jina/blob/master/.github/pull_request_template.md), [mindsdb](https://github.com/mindsdb/mindsdb/blob/staging/.github/pull_request_template.md). So, I thought to change it for our repo too.

Reply:
> I think both are okay but I felt like `this PR closes this issue` sounds better + I see that most of the repos use `Closes` or `Fixes` in the PR template [pandas](https://github.com/pandas-dev/pandas/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [dask](https://github.com/dask/dask/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [pytorch](https://github.com/pytorch/pytorch/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [jina](https://github.com/jina-ai/jina/blob/master/.github/pull_request_template.md), [mindsdb](https://github.com/mindsdb/mindsdb/blob/staging/.github/pull_request_template.md). So, I thought to change it for our repo too.

those two key words are there to let github know which issue to link to in the Development section.

Reply:
> > I think both are okay but I felt like `this PR closes this issue` sounds better + I see that most of the repos use `Closes` or `Fixes` in the PR template [pandas](https://github.com/pandas-dev/pandas/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [dask](https://github.com/dask/dask/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [pytorch](https://github.com/pytorch/pytorch/blob/main/.github/PULL_REQUEST_TEMPLATE.md), [jina](https://github.com/jina-ai/jina/blob/master/.github/pull_request_template.md), [mindsdb](https://github.com/mindsdb/mindsdb/blob/staging/.github/pull_request_template.md). So, I thought to change it for our repo too.
> 
> By the way, those two key words are there to let github know which issue to link to in the Development section.

Yes, sir. I see that both `close` and `closes` can be used as key words as specified by [github here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue), and since most of the big organizations are using `Closes` or `Fixes` I thought to change ours also to `Closes` 😄.

Reply:
Yeah `closes` sounds good as well! We could use `closes` instead, what do you think @a0m0rajab ?

Reply:
@a0m0rajab could you please take a look? Thanks!

Reply:
The output of this file is something like this picture: 
![image](https://github.com/unifyai/ivy/assets/18273833/2f19fb2d-eb28-49f6-93db-8b183fb5c0ad)

So either keyword is okay, since we are not using a full sentence there.

Reply:
sounds good, I'll resolve this conversation now!